src/NHSD.BuyingCatalogue.Organisations.Api/Models/Results/Result.cs

```diff
@@ -29,11 +29,21 @@ public static Result Success()
             return _success;
         }
 
+        public static Result<T> Success<T>(T value)
+        {
+            return new Result<T>(true, default, value);
+        }
+
         public static Result Failure(IEnumerable<ErrorMessage> errors)
         {
             return new Result(errors);
         }
 
+        public static Result<T> Failure<T>(IEnumerable<ErrorMessage> errors)
```

Comment:
Yeah good shout. An extension did cross my mind, but a collection class could be better, just wondering how it would fit in as a read only collection. Will keep an eye on more list equality.